### ISSUE TITLE
Make the error code more friendly

### ISSUE THREAD

**Issue by Amber1990Zhang** [vesoft-inc/nebula#1974]
I want to revoke a role from the user, if I choose the wrong role, nebula throws me an error, but the error message is not easy to understand. I prefer error message like **Wrong role used.**

```
(user@127.0.0.1:3699) [gods]> SHOW ROLES in gods
=======================
| Account | Role Type |
=======================
| wumin   | DBA       |
-----------------------
Got 1 rows (Time spent: 1.272/5.065 ms)

Thu Mar 26 02:58:57 2020

(user@127.0.0.1:3699) [gods]> REVOKE ROLE guest on gods from wumin
[ERROR (-8)]: Revoke failed: Unknown code -42.
Thu Mar 26 02:59:30 2020

(user@127.0.0.1:3699) [gods]> REVOKE ROLE dba on gods from wumin
Execution succeeded (Time spent: 7.838/9.004 ms)
```

**Comment by bright-starry-sky** [comment]
(root@127.0.0.1:6999) [(none)]> CHANGE PASSWORD user2 FROM "pwd" TO "newpwd"
[ERROR (-8)]: Unknown code -41
Thu Mar 26 12:16:55 2020

**Comment by bright-starry-sky** [comment]
The error message fixed as below , I will submit a pr at later.

(root@127.0.0.1:6999) [(none)]> REVOKE DBA ON user_space FROM user1
[ERROR (-8)]: Revoke failed: Improper role.
Thu Mar 26 14:31:33 2020

(root@127.0.0.1:6999) [(none)]> CHANGE PASSWORD user2 FROM "pwd" TO "newpwd"
[ERROR (-8)]: Invalid password
Thu Mar 26 13:55:18 2020

**Comment by bright-starry-sky** [comment]
fixed via #1980
